backend/src/utils/image-utils.ts

```diff
@@ -0,0 +1,97 @@
+import { bucket } from '../firebase';
+import { getNetIDFromEmail } from './memberUtil';
+import { NotFoundError } from './errors';
+
+/**
+ * Sets image for member
+ * @param name - the name of the image
+ * @returns a Promise to the signed URL to the image file
+ */
+export const setImage = async (name: string): Promise<string> => {
+  const file = bucket.file(`${name}.jpg`);
```

Comment:
on a side note - probably better off including the file extension in the actual file path itself instead of assuming they're all `.jpg`